app/assets/stylesheets/_mini_groove_song_card.css.sass

```diff
@@ -0,0 +1,25 @@
+.songcard
+	width: 90%
+	height: 100px
+	border: 1px black solid 
+	border-radius: 5px
+	margin: 10px auto
+	overflow: hidden
+	background-color: #7879BD
+
+	
+.songinfo
+	display: inline
+	margin-left: 30px
+	margin-top: 10px
+
+	
+.coverart
+	float: left
+	clear: both
+
+	
+	img
+		height: 100px
+		overflow: hidden
```

Comment:
:+1: to using SASS partials so all CSS related to one element is together! :confetti_ball: